inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -11,7 +13,7 @@ export class CanvasGraphService {
     private _width: number;
     private _height: number;
     public readonly datasets: IPerfDataset[];

```

Comment:
[NIT] might be more readable to split privates and publics

Reply:
Oh woah you're right, I didn't notice I thing during a rebase I must've misplaced the datasets variable! good callout!

Reply:
Done!